server/src/pages/homepage/view.js

```diff
@@ -9,14 +9,14 @@ class Head extends React.Component {
       <reactruntime.HeadTemplate {...this.props}>
         <script src={this.props.staticLink("/static/js/homepage-bundle.js")} async></script>
         <meta name="viewport" content="width=320, initial-scale=1.0, maximum-scale=1.0, user-scalable=0" />
-        <meta name="description" content="Share anything on the web with anyone using Firefox Screenshots." />
+        <meta name="description" content="Share anything on the web with anyone using Firefox Screenshots." /> <!-- todo l10n: homePageDescription -->
```

Comment:
I've come across [react-intl][1] and [react-globalize][2] before. I haven't used them extensively in my projects. I'd like to test them in depth and write a comparison with fluent-react in the future. They both use MessageFormat for storing translations which isn't supported out of the box by Pontoon and would require some kind of build-time transformation to get into Pontoon.

If you're worried about attributes, Pontoon already supports some FTL and its support is constantly improving :)

[1]: https://github.com/yahoo/react-intl
[2]: https://github.com/globalizejs/react-globalize